src/tests/test_deps.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from app.api import crud
+from app.api.deps import get_current_user, get_current_device
```

Comment:
You may look at the Unit test refactoring in https://github.com/pyronear/pyro-api/pull/33 to stay in the same spirit.

Reply:
Not done yet:
I agree; but I am struggling a little bit to monkey-patch the users and devices tables in this module (test_deps); but I will take this comment into account in my next PR